src/types.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import type {
+    NpmLogLevels,
+    Logger as WinstonLogger,
+    Info as WinstonInfo,
+} from "winston";
+
+export type Info = {
+    ...WinstonInfo<NpmLogLevels>,
+    durationMs: number,
+    ...
+};
+
+export type LogLevel = $Keys<NpmLogLevels>;
+export type Logger = WinstonLogger<NpmLogLevels>;
+
+/* eslint-disable flowtype/no-dupe-keys */
```

Comment:
Is this still relevant? It's not clear what this is a duplicate of?

Reply:
We might want to add `"eslint-comments/no-unused-disable": "error",`.

Reply:
Hmm, I'll look into this as I thought I had that lint rule.

Reply:
OK, I added the rule, but this is still a required suppression. The `flowtype/no-dupe-keys` rule appears to be incorrectly flagging the `get` keyword as as key and getting upset that it's repeated. I've added a comment to explain that.

Reply:
I've raised an issue: https://github.com/gajus/eslint-plugin-flowtype/issues/431